editor/src/components/filebrowser/filebrowser.tsx

```diff
@@ -108,7 +120,21 @@ function collectFileBrowserItems(
       }
     }
   })
-  return fileBrowserItems
+  return fileBrowserItems.map((f) => {
```

Comment:
It looks like we could roll this into the above `walkContentsTree` call (unless I've missed something)?